deployments/admission-controllers/scheduler/generate-signed-ca.sh

```diff
@@ -24,8 +24,8 @@ if [ ! -f ${CONF_FILE} ]; then
 fi
 
 tmpdir="$1"
-service=`cat ${CONF_FILE} | grep ^service | cut -d "=" -f 2`
-namespace=`cat ${CONF_FILE} | grep ^namespace | cut -d "=" -f 2`
+service=$SERVICE
```

Comment:
please add a check here in case environment variables are missing here

Reply:
OK